toolkit/docs/building/building.md

```diff
@@ -199,32 +185,32 @@ sudo make image CONFIG_FILE=./imageconfigs/core-legacy.json REBUILD_TOOLS=y DOWN
 # To build a Mariner VHDX Image (VHDX folder ../out/images/core-efi)
 sudo make image CONFIG_FILE=./imageconfigs/core-efi.json REBUILD_TOOLS=y DOWNLOAD_SRPMS=y
 
-# To build a Mariner ISO Image (ISO folder: ../out/images/full)
-sudo make iso CONFIG_FILE=./imageconfigs/full.json REBUILD_TOOLS=y DOWNLOAD_SRPMS=y
-
 # To build a Mariner Contianer Image (Container Folder: ../out/images/core-container/*.tar.gz
 sudo make image CONFIG_FILE=./imageconfigs/core-container.json REBUILD_TOOLS=y DOWNLOAD_SRPMS=y
 ```
 
-### ISOs
+### ISO Images
+ISOs are bootable images that install CBL-Mariner to either a physical or virtual machine.  The installation process can be manually guided through user prompting, or automated through unattended installation.
+
+NOTE: ISOs require additional packaging and build steps (such as the creation of a separate `initrd` installer image used to install the final image to disk).  These additional resources are stored in the toolkit/resources/imagesconfigs folder.
 
-ISO installers can be built with:
 
+The following builds an ISO with an interactive UI and selectable image configurations.
 ```bash
-# Build out/images/developer_iso/*.iso from remote components
-sudo make iso -j$(nproc) CONFIG_FILE=./resources/imageconfigs/developer_iso/developer_iso.json
+# To build a Mariner ISO Image (ISO folder: ../out/images/full)
+sudo make iso CONFIG_FILE=./imageconfigs/full.json REBUILD_TOOLS=y DOWNLOAD_SRPMS=y
 ```
 
-To create an unattended ISO installer (no interactive UI) use `UNATTENDED_INSTALLER=y`:
+To create an unattended ISO installer (no interactive UI) use `UNATTENDED_INSTALLER=y` and with a CONFIG_FILE that only specifies a single SystemConfig.
 
 ```bash
-# Build out/images/developer_iso/*.iso from remote components with unattended installer
-sudo make iso -j$(nproc) CONFIG_FILE=./resources/imageconfigs/developer_iso/developer_iso.json UNATTENDED_INSTALLER=y
+# Build the standard ISO with unattended installer
+sudo make iso -j$(nproc) CONFIG_FILE=./imageconfigs/core-legacy.json UNATTENDED_INSTALLER=y
```

Comment:
>UNATTENDED_INSTALLER=y [](start = 69, length = 22)

I think we still need `REBUILD_TOOLS=y DOWNLOAD_SRPMS=y`.

Reply:
REBUILD_TOOLS=y makes sense.  Fixed.  Seems to me DOWNLOAD_SRPMS would be a bug at that point.  All the SRPMS should already have been built.

Reply:
You're right. In that case we should remove `DOWNLOAD_SRPMS=y` from the snippet above.

---
In reply to: [546982669](https://github.com/microsoft/CBL-Mariner/pull/476#discussion_r546982669) [](ancestors = 546982669)